github/Repository.py

```diff
@@ -2325,15 +2325,16 @@ def get_forks(self):
     def create_fork(self, organization=github.GithubObject.NotSet):
         """
         :calls: `POST /repos/{owner}/{repo}/forks <https://docs.github.com/en/rest/reference/repos#forks>`_
-        :param organization: string or "none" or "*"
+        :param organization: :class:`github.Organization.Organization` or string
         :rtype: :class:`github.Repository.Repository`
         """
-        assert organization is github.GithubObject.NotSet or isinstance(
-            organization, str
-        ), organization
         post_parameters = {}
-        if organization is not github.GithubObject.NotSet:
+        if isinstance(organization, Organization.Organization):
```

Comment:
Surely this is NameError? We should also add a test calling create_fork with an object.